### PR TITLE
Optional strict parameter for parsing csv feeds 

### PR DESCRIPTION
**Pull request by jakehoskins**

### CODE REVIEW

File: pyalgotrade/barfeed/googlefeed.py [hunk 1]
```diff
@@ -79,20 +81,33 @@ def getFieldNames(self):
...
         return ","
 
-    def parseBar(self, csvRowDict):
-        dateTime = self.__parseDate(csvRowDict["Date"])
-        close = float(csvRowDict["Close"])
-        open_ = float(csvRowDict["Open"])
-        high = float(csvRowDict["High"])
-        low = float(csvRowDict["Low"])
-        volume = float(csvRowDict["Volume"])
+    def parseBar(self, csvRowDict, rejectInvalid=True):
```
Comment (jakehoskins) [inline pyalgotrade/barfeed/googlefeed.py]: remove

**Comment by gbeced** [comment]
I will pull this in with some changes.

File: pyalgotrade/barfeed/csvfeed.py [hunk 2]
```diff
@@ -94,11 +94,12 @@ class BarFeed(membf.BarFeed):
         This is a base class and should not be used directly.
     """
 
-    def __init__(self, frequency, maxLen=None):
+    def __init__(self, frequency, maxLen=None, rejectInvalid=True):
```
Comment (gbeced) [inline pyalgotrade/barfeed/csvfeed.py]: This parameter is unsued.

**Comment by gbeced** [comment]
I am working on an alternative implementation that is more simple than this, and implements the feature for any csvfeed derived class, and I found out that Google Finance changed their API. I'll probably address that in a separate branch.

**Comment by gbeced** [comment]
I'm closing this pull request since I included the same functionality with a simpler implementation and also included test cases:
https://github.com/gbeced/pyalgotrade/commit/b2667acae171d7abe0ad857c9fe466169c4713a0

**Pull request closed without merging**